src/components/Challenges/CodeChallenge/CodeChallenge.tsx

```diff
@@ -46,6 +46,17 @@ export const CodeChallenge = ({
 
   useEffect(resetChallenge, [id]);
 
+  // passes test when control & enter keys are pressed together
+  useEffect(() => {
```

Comment:
Can you add cleanup to the hook as well? Check out https://reactjs.org/docs/hooks-effect.html to see what I mean regarding cleanup